src/lib/apiTypesV2/BatchQuery.h

```diff
@@ -51,7 +51,7 @@ class BatchQuery
   BatchQuery();
   ~BatchQuery();
 
-  std::string    check(ConnectionInfo* ciP, RequestType requestType);
+  //std::string    check(ConnectionInfo* ciP, RequestType requestType);
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/2679/commits/7bcc3414bae24dbce4d999882838acc219088515